src/rtcModule/webrtc.cpp

```diff
@@ -2359,9 +2392,24 @@ void Call::onClientLeftCall(Id userid, uint32_t clientid)
         {
             mManager.launchCallRetry(mChat.chatId(), sentAv(), false);
         }
+    }
+    else if (mState >= kStateInProgress)
+    {
+        if (userid == mManager.mKarereClient.myHandle())
```

Comment:
Perhaps we need to add `&& !chat.isGroup()` to the `if` above.

Reply:
That part of code was added by merge. It's not necessary